src/log/log_functions.c

```diff
@@ -119,6 +119,11 @@ int global_id = 0;
 __thread int id = -1;
 __thread log_file_t *lbuffer_thr = NULL;
 
+/* Variables for log rotate */
+
+const int MAX_OLD_LOGFILES = 5;
+const int MAX_LOGFILE_SIZE = 200 * 1000 * 1000; /* 200MB */
```

Comment:
200 * 1024 * 1024 ideally

Reply:
so max space should also be a parameter we should decide or make it a default value. What you think should we put a default value to it? I tried 200 * 1024 * 1024 but ls -lh is showing more size in MB, so changed it to 200 * 1000 * 1000. Anyways I will make the change, as it is the correct one.

Reply:
After discussing with Chandra, following is decided.
Each log file max size 100MB By default. It is a configurable parameter as follows
# Maximum Logfile Size, Default 100MB
 Max_Logfile_Size = 100000000;
User will be responsible for deciding number of log files and max log file size depending on disk available.

We will have one current log and 4 backedup for each log file(num_log_file)

-rw-r--r-- 1 mapr mapr 3.0M Aug 16 16:58 nfs4server.log-0
-rw-r--r-- 1 mapr mapr 9.8M Aug 16 16:58 nfs4server.log-0.1
-rw-r--r-- 1 mapr mapr 9.6M Aug 16 16:58 nfs4server.log-0.2
-rw-r--r-- 1 mapr mapr 9.8M Aug 16 16:58 nfs4server.log-0.3
-rw-r--r-- 1 mapr mapr 9.8M Aug 16 16:58 nfs4server.log-0.4

-rw-r--r-- 1 mapr mapr 2.3M Aug 16 16:58 nfs4server.log-1
-rw-r--r-- 1 mapr mapr 9.8M Aug 16 16:58 nfs4server.log-1.1
-rw-r--r-- 1 mapr mapr 9.7M Aug 16 16:58 nfs4server.log-1.2
-rw-r--r-- 1 mapr mapr 9.8M Aug 16 16:58 nfs4server.log-1.3
-rw-r--r-- 1 mapr mapr 9.7M Aug 16 16:58 nfs4server.log-1.4

Above configuration has 10MB as max log file size.
Updated and tested code with this.